crates/fraud_proofs/global_merkle_root/storage/src/update.rs

```diff
@@ -63,7 +75,7 @@ use fuel_core_types::{
     },
 };
 
-pub trait UpdateMerkleizedTables {
+pub trait UpdateMerkleizedTables: Sized {
```

Comment:
I don't think this is needed, and I believe this is a merge artifact since I removed this redundant bound in the base PR previously.

```suggestion
pub trait UpdateMerkleizedTables: {
```